biothings_client/cache/storage.py

```diff
@@ -0,0 +1,260 @@
+"""
+Custom cache storage handling for the biothings-client
+"""
+
+from pathlib import Path
+from typing import Optional, Union
+import logging
+import sqlite3
+
+import anysqlite
+import hishel
```

Comment:
if we make these two packages optional in `pyprojects.toml`, we should add a import check here, like we did for requests_cache before.